### PR TITLE
Update manual install docs & config errors

### PR DESCRIPTION
**Pull request by davidmehren**
### Component/Part
documentation & logging

### Description
Starting with 1.7, releases include the frontend bundle.
This PR updates the docs accordingly.

Additionally, remaining mentions of `config.js` in log statements were replaced by `config.json`.

### Steps

- [x] Fixed implementation
- [x] Added / updated documentation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/master/CONTRIBUTING.md) and signed-off my commits to accept the DCO.

### CODE REVIEW

**Review by InnayTool — APPROVED** [review]

File: docs/setup/manual-setup.md [hunk 1]
```diff
@@ -3,41 +3,31 @@
...
   Be aware of older MySQL and MariaDB versions which sometimes use shorter representations of UTF-8 than 4 bytes.
   This can break if symbols with more bytes are used.
   You can use `alter database <DBNAME> character set utf8mb4 COLLATE utf8mb4_unicode_ci` to be on the safe side.
-
-- npm (and its dependencies, [node-gyp](https://github.com/nodejs/node-gyp#installation))
-
-- yarn
-
+- Npm (and its dependencies, [node-gyp](https://github.com/nodejs/node-gyp#installation))
+- Yarn
```
Comment (ErikMichelson) [inline docs/setup/manual-setup.md:13]: Why did you capitalize the software names here?

Comment (davidmehren) [reply docs/setup/manual-setup.md:13]: Because all other list items are also capitalized :D

File: docs/setup/manual-setup.md [hunk 2]
```diff
@@ -3,41 +3,31 @@
...
-3. Enter the directory and type `bin/setup`, which will install npm dependencies and create configs.
-
+2. Download a [release](https://github.com/hedgedoc/hedgedoc/releases) tarball and extract it.
+3. Enter the directory and type `bin/setup`, which will install the dependencies and create configs.
 4. Modify the file named `config.json` or configure HedgeDoc through environment variables which will overwrite the configs, see docs [here](https://github.com/hedgedoc/hedgedoc/blob/master/docs/configuration.md).
-
-5. Build front-end bundle by `yarn run build` (use `yarn run dev` if you are in development)
-
-6. Modify the file named `.sequelizerc`, change the value of the variable `url` with your db connection string. For example:
+5. **Skip this step for release 1.7 and newer!**
```
Comment (SISheogorath) [inline docs/setup/manual-setup.md:24]: I would rather say that one **can** skip this, but not that they should.

File: docs/setup/manual-setup.md [hunk 3]
```diff
@@ -3,41 +3,31 @@
...
 
 ## Instructions
 
 1. Check if you meet the [requirements at the top of this document](#requirements-on-your-server).
-
-2. Clone this repository (preferred) or download a release and unzip it.
-
-3. Enter the directory and type `bin/setup`, which will install npm dependencies and create configs.
-
+2. Download a [release](https://github.com/hedgedoc/hedgedoc/releases) tarball and extract it.
```
Comment (SISheogorath) [inline docs/setup/manual-setup.md:21]: I would add an explicit node that one can either clone the repository using the `-b` parameter to get the stable version or use a release.

**Comment by davidmehren** [comment]
@SISheogorath I updated the PR to address your comments :)

File: docs/setup/manual-setup.md (hunk 3 above)
Comment (davidmehren) [reply docs/setup/manual-setup.md:21]: Fixed in the following line

**Review by ErikMichelson — APPROVED** [review]

**Review by SISheogorath — APPROVED** [review]

**Pull request merged**